tests/utils.py

```diff
@@ -93,17 +101,35 @@ def __init__(self, dataset_name, test_fraction):
             N = 1000
             self.X = np.random.random(size=(N, 200))
             self.y = np.random.randint(3, size=(N,))
+        elif dataset_name == "classification_rnd_w_missing_values":
+            self.name = "train_model_classification_rnd_w_missing_values"
+            N = 1000
+            self.X = np.random.random(size=(N, 200)) - 0.5
```

Comment:
I believe `1000x200` is too much for this test. `1000x200` was used in random data to test very large trees: #105.